### PR TITLE
feat: Add add-order to tickets route

### LINKED ISSUE #114: Add add-order to ticket route(UI only)
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

Add add-order as sub route of tickets.

screenshot from 2017-06-02 13-42-33

- Located at `tickets/add-order`

Please have a look at #97, before starting the work on the issue.

### PR DESCRIPTION
**Pull request by utkarshnath**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

Fixes #114

### CODE REVIEW

File: app/templates/events/view/tickets/add-order.hbs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    <tr>
+      <th>{{t 'Ticket Type'}}</th>
+      <th class="right aligned">{{t 'Price  '}}(US$)</th>
+      <th class="right aligned">{{t 'Quantity'}}</th>
+      <th class="right aligned">{{t 'Item Total'}}</th>
+    </tr>
+    <tr>
+      <td>{{t 'Free'}}</td>
+      <td class="right aligned">{{t 'US$ 0.00	'}}</td>
+      <td class="right aligned">{{input value=quantity_item1 type="number" min="0"}}</td>
```
Comment (geekyd) [inline app/templates/events/view/tickets/add-order.hbs]: Please use single quotes inside helpers, follow https://github.com/fossasia/open-event-frontend/blob/development/docs/style-guide.md

File: app/templates/events/view/tickets/add-order.hbs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    </tr>
+  </thead>
+</table>
+<div class="ui grid">
+  <div class="row " style="padding-top:20px">
+    <div class="two wide column right aligned">
+      <h3 class="ui header">{{t 'Grand Total'}}</h3>
+    </div>
+    <div class="two wide column right aligned">
+      {{input value=total_item1 type="number" min="0"}}
```
Comment (geekyd) [inline app/templates/events/view/tickets/add-order.hbs]: Please use single quotes inside helpers, follow https://github.com/fossasia/open-event-frontend/blob/development/docs/style-guide.md

**Comment by utkarshnath** [comment]
@niranjan94 @geekyd updated

**Comment by geekyd** [comment]
@utkarshnath Please resolve the conflicts & provide link of application for testing.

**Comment by utkarshnath** [comment]
https://young-wildwood-91762.herokuapp.com/ plz check

File: app/templates/events/view/tickets/add-order.hbs [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    <div class="two wide column right aligned">
+      <h3 class="ui header">{{t 'Grand Total'}}</h3>
+    </div>
+    <div class="two wide column right aligned">
+      {{input value=total_item1 type='number' min='0'}}
+    </div>
+  </div>
+</div>
+<div class="row" style="padding-top:20px">
+  {{#ui-dropdown class='labeled icon button'}}
```
Comment (geekyd) [inline app/templates/events/view/tickets/add-order.hbs]: Please place dropdown & Continue button in same line

File: app/templates/events/view/tickets/add-order.hbs [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+      <td>{{t 'Free'}}</td>
+      <td class="right aligned">{{t 'US$ 0.00'}}</td>
+      <td class="right aligned">{{input value=quantity_item1 type='number' min='0'}}</td>
+      <td class="right aligned">{{input value=total_item1 type='number' min='0'}}</td>
+    </tr>
+  </thead>
+</table>
+<div class="ui grid">
+  <div class="row" style="padding-top:20px">
+    <div class="two wide column right aligned">
```
Comment (geekyd) [inline app/templates/events/view/tickets/add-order.hbs]: Make this `sixteen column wide` that will align it to right.

**Comment by utkarshnath** [comment]
@geekyd @niranjan94 plz review
https://young-wildwood-91762.herokuapp.com/

File: app/templates/events/view/tickets/add-order.hbs [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+<div class="ui grid stackable">
+  <div class="row">
+    <div class="sixteen wide column right aligned">
+      <h4 class="ui header">{{t 'Grand Total'}}</h4>
+    </div>
+    <div class="sixteen wide column right aligned">
+      <h5 class="ui header">200 $</h5>
+    </div>
+  </div>
+  <div class="row" style="padding-top:20px">
```
Comment (geekyd) [inline app/templates/events/view/tickets/add-order.hbs]: Please add space after `:`

File: app/templates/events/view/tickets/add-order.hbs [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+      <th>{{t 'Ticket Type'}}</th>
+      <th class="right aligned">{{t 'Price '}}(US$)</th>
+      <th class="right aligned">{{t 'Quantity'}}</th>
+      <th class="right aligned">{{t 'Item Total'}}</th>
+    </tr>
+    <tr>
+      <td>{{t 'Free'}}</td>
+      <td class="right aligned">{{t 'US$ 0.00'}}</td>
+      <td class="right aligned">{{input value=quantity_item1 type='number' min='0'}}</td>
+      <td class="right aligned">{{input value=total_item1 type='number' min='0'}}</td>
```
Comment (geekyd) [inline app/templates/events/view/tickets/add-order.hbs]: Use `type='text'`, This represents the total of the item i.e (price * number of items).

File: app/templates/events/view/tickets/add-order.hbs [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+    <tr>
+      <th>{{t 'Ticket Type'}}</th>
+      <th class="right aligned">{{t 'Price '}}(US$)</th>
+      <th class="right aligned">{{t 'Quantity'}}</th>
+      <th class="right aligned">{{t 'Item Total'}}</th>
+    </tr>
+    <tr>
+      <td>{{t 'Free'}}</td>
+      <td class="right aligned">{{t 'US$ 0.00'}}</td>
+      <td class="right aligned">{{input value=quantity_item1 type='number' min='0'}}</td>
```
Comment (geekyd) [inline app/templates/events/view/tickets/add-order.hbs]: Update Total & grand total when we change quantity of ticket. check how this was implemented in `ticket-list` component in the public route.

**Comment by utkarshnath** [comment]
@niranjan94 @geekyd plz review
https://young-wildwood-91762.herokuapp.com/events/963d3ba5/tickets/add-order

**Comment by CosmicCoder96** [comment]
@utkarshnath Please resolve the conflicts so I can merge it, thanks!

**Comment by utkarshnath** [comment]
@CosmicCoder96  @geekyd  
updated plz merge

**Pull request merged**